cogs/info.py

```diff
@@ -16,6 +16,44 @@ def __init__(self, bot):
         self.process = psutil.Process(os.getpid())
 
     ## Proof of concept for use with @story{9}
+    @commands.command()
+    async def trivia (self, ctx, arg1 = "", arg2 = ""):
+
+        # *** BEGIN ***
```

Comment:
This commenting style is unfamiliar to me. Does it have special meaning. Is there some standard it follows, or is it just a project convention?

Reply:
No, these do not follow any convention or anything else special. I just wrote them like that to make them easy to see and separate sections of the code. I felt it made it easier to scan through quickly.